lbrynet/daemon/Daemon.py

```diff
@@ -3082,29 +3045,264 @@ def jsonrpc_stream_availability(self, uri, search_timeout=None, blob_timeout=Non
+                }
+                for chain_name in address_changes:
+                    chain = getattr(account, chain_name)
+                    for attr, value in address_changes[chain_name].items():
+                        if value is not None:
+                            setattr(chain, attr, value)
+                            change_made = True
+            if change_made:
+                wallet.save()
+
+        if default:
+            wallet.accounts.remove(account)
+            wallet.accounts.insert(0, account)
+            wallet.save()
+
+        result = account.to_dict()
+        result.pop('certificates', None)
+        result['is_default'] = wallet.accounts[0] == account
+        return result
+
+    @requires("wallet")
+    def jsonrpc_balance(self, account_name=None, confirmations=6, include_reserved=False,
+                        include_claims=False):
+        """
+        Return the balance of an individual account or all of the accounts.
+
+        Usage:
+            balance [<account_name>] [--confirmations=<confirmations>]
+                    [--include_reserved] [--include_claims]
+
+        Options:
+            --account=<account_name>        : (str) If provided only the balance for this
+                                                    account will be given
+            --confirmations=<confirmations> : (int) required confirmations (default: 6)
+            --include_reserved              : (bool) include reserved UTXOs (default: false)
+            --include_claims                : (bool) include claims, requires than a
+                                                     LBC account is specified (default: false)
+
+        Returns:
+            (map) balance of account(s)
+        """
+        if account_name:
+            for account in self.wallet.accounts:
+                if account.name == account_name:
+                    if include_claims and not isinstance(account, LBCAccount):
+                        raise Exception(
+                            "'--include-claims' requires specifying an LBC ledger account. "
+                            "Found '{}', but it's an {} ledger account."
+                            .format(account_name, account.ledger.symbol)
+                        )
+                    args = {
+                        'confirmations': confirmations,
+                        'include_reserved': include_reserved
+                    }
+                    if include_claims:
+                        args['include_claims'] = True
+                    return account.get_balance(**args)
+            raise Exception("Couldn't find an account named: '{}'.".format(account_name))
+        else:
+            if include_claims:
+                raise Exception("'--include-claims' requires specifying an LBC account.")
+            return self.wallet.get_balances(confirmations)
+
+    @requires("wallet")
+    def jsonrpc_max_address_gap(self, account_name):
```

Comment:
`wallet_max_address_gap`

Reply:
Is there a significant benefit to adding `wallet` to this commands name? Seems like it just forces users to type more on the command line...